### DISCUSSION TITLE
[Question] Attaching files?

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [hay-kot/homebox#233] [Q&A]
Hello!

I am looking for an inventory system to manage the tech stuff I have here at my home. However, I would also like to be able to attach files.

While trying out the demo, I couldn't figure out how to apply those attachments to an item. In my case, I want to store zips of drivers and related utility programs to some of the devices I have (i.e. drivers, flash tools, ...).

How do I do that - or is that just disabled in the demo?

Thank you and kind regards,
Ingwie

**Comment by hay-kot** [comment]
You can add attachments under the edit page in the attachments section

These will show up on your items page

![CleanShot 2023-01-20 at 10 00 16@2x](https://user-images.githubusercontent.com/64056131/213783921-12ded434-b006-40ec-b842-135c7e08d1ff.png)

**Reply by IngwiePhoenix** [reply]
Both in the demo and when I run it locally (on an arm64 sbc), this section doesnt show up .

The version that runs when I run it via docker is reported as: `Version: v0.6.0 ~ Build: 6af048dc93a303d6b62b022664006d84897c5ebe`

Here's a screenshot:
![image](https://user-images.githubusercontent.com/2893716/213852865-de8ce3c9-8a24-407f-83ae-325f7dc1d517.png)

**Reply by hay-kot** [reply]
There should be a toggle at the top to show additional fields.

**Reply by IngwiePhoenix** [reply]
Oh... Welp, that one's on me, completely oversaw that. Can I make the full view the default somehow?

And thanks for making this tool, its awesome :)

**Reply by hay-kot** [reply]
It remembers your settings, whatever you left it at last should be the default view.

**Comment by stegl** [comment]
hello everyone

in attachments now we have to select type of a file.. is this bug or is it meant to be like this? 
![autotype](https://user-images.githubusercontent.com/65857447/219873040-1caf1377-3c64-4c3e-9f5c-4b599c71f36a.png)

**Reply by hay-kot** [reply]
It's always been like this. We don't do anything to auto detect the type of file, though we could try to.

**Reply by stegl** [reply]
Maybe just for pictures?
